src/js/wrap.cpp

```diff
@@ -189,14 +190,104 @@ namespace ccf::js
     JS_FreeRuntime(rt);
   }
 
+  static KVMap::Handle* _get_map_handle(
+    js::Context& jsctx, JSValueConst this_val)
+  {
+    auto map_name =
+      jsctx.to_str(JS_GetPropertyStr(jsctx, this_val, "_map_name"));
```

Comment:
Might be good to wrap this_val, and then JS_CHECK_EXC() the rv of this_wrapped_val["_map_name"], just in case there's a problem retrieving it. There probably won't be in this case, but it's prudent and will give better errors.